types/gestalt/index.d.ts

```diff
@@ -238,7 +239,7 @@ https://pinterest.github.io/gestalt/#/Flyout
 */
 
 export interface FlyoutProps {
-    anchor: React.RefObject<any>;
+    anchor: any; // ideally a HTMLAnchorElement
```

Comment:
so should it then read?:
```ts
export interface FlyoutProps {
    anchor?: React.RefObject<HTMLElement>;// ideally a HTMLAnchorElement
```

Reply:
No - flyout doesn't want a `React.RefObject<HTMLElement>`, it wants `HTMLElement`. 

i.e. instead of passing `<Flyout anchor={ref} />` you now need to pass `<Flyout anchor={ref.current} />`